libs/base/Data/Vect/Quantifiers.idr

```diff
@@ -195,6 +205,19 @@ namespace All
   traversePropertyRelevant f [] = pure []
   traversePropertyRelevant f (x :: xs) = [| f _ x :: traversePropertyRelevant f xs |]
 
+  public export
+  tabulate : {n : _} ->
+             {0 xs : Vect n _} ->
+             (f : (ix : Fin n) -> p (ix `index` xs)) ->
+             All p {n} xs
+  tabulate {xs = []} f = []
+  tabulate {xs = _ :: _} f = f FZ :: tabulate (\ix => f (FS ix))
+
+  public export
+  (++) : (axs : All p xs) -> (ays : All p ys) -> All p (xs ++ ys)
```

Comment:
Why not adding the same functions to `Data.List.Quantifiers.All`?

Reply:
You mean just these two or all of what this PR adds?

Reply:
I meant at least these two, but I think all of them worth adding